scss/styles.scss

```diff
@@ -0,0 +1,95 @@
+$vars-nav-height: 90px;
```

Comment:
the prefix `vars-` is also redundant just say `$nav-height`, etc. 

I would also move this to another file named `sizes.scss` so we can keep everything separated and in small files.